python/private/common/entrypoint.tmpl.py

```diff
@@ -0,0 +1,21 @@
+import runpy
+import sys
+from pathlib import Path
+
+from python.runfiles import runfiles
+
+RUNFILES = runfiles.Create()
+RUNFILES_DIR = Path(RUNFILES.EnvVars()["RUNFILES_DIR"])
+MAIN = RUNFILES_DIR / "%MAIN_REPO%/%MAIN_SHORT_PATH%"
+
+IMPORTS = %IMPORTS%
+
+# Work around sys.path[0] escaping the sandbox by deleting it.
+# See https://github.com/bazelbuild/rules_python/issues/382 for more info.
+# TODO(phil): How do we distinguish between safe-path Python and non-safe-path
```

Comment:
For inspiration, see: https://github.com/bazelbuild/rules_python/blob/627830e0756e6f7c09f2a95561963121dff9c166/python/private/py_console_script_gen.py#L50

Reply:
Super cool. Thank you!